string stop_id
string arrival_time
string departure_time
GpsCoordinat position
